spec/components/schemas/entityperiod.yaml

```diff
@@ -3,7 +3,7 @@ properties:
   object:
     type: string
     enum: ["entityperiod"]
-  id:
```

Comment:
I am unsure of why we would want to change this property. A UUID is an 'id'.

Reply:
We are going with the property name 'uuid'. It requires less effort with DRF and doesn't the name isn't that important.